src/main/pages/che-7/end-user-guide/proc_network-requirement.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// network-requirement
+
+[id="network-requirement_{context}"]
+= Network requirement
+
+Enabling the WebSocket protocol is critical for the proper functionality of {prod-short} IDE.
+
+While the WebSocket protocol itself is unaware of proxy servers and firewalls, HTTP servers can share their default HTTP and HTTPS ports with a WebSocket server.
+
+* HTTP: port 80
+* HTTPS: port 433
+
+Some proxy servers operate with WebSockets by default, but others prevent WebSockets from working correctly, which causes the connection to fail.
+
+In some cases, the proxy server requires the additional configuration, and the specific proxy servers need an upgrade, which allows WebSockets support.
```

Comment:
This doesn't look like a procedure. What needs to be done here? What are the procedure steps? What are we configuring? HTTP server? Proxy server?

Reply:
no we are just checking if the problem is about websocket or not, this seems to be possible with the site provided https://websocketstest.com/[realtime web test]. This is the procedure. We don't provide procedure of proxy that we don't know anything about. Any organisation has their own proxies with different brand ... it is out of our scope.